### DISCUSSION TITLE
Is there the way to find out Sprite-Spell info?

### DISCUSSION THREAD

**Discussion by 1Sam** [fraxiinus/ReplayBook#190] [Feature Request]
http://ddragon.leagueoflegends.com/cdn/12.10.1/data/en_US/summoner.json
http://ddragon.leagueoflegends.com/cdn/12.10.1/img/sprite/spell0.png
http://ddragon.leagueoflegends.com/cdn/12.10.1/img/spell/SummonerFlash.png

Coud you teach me how to get the Info like Smite and Flash?

What does it  mean 'SUMMON_SPELL1_CAST, ?
"STAT_PERK_0":"5005","STAT_PERK_1":"5008","STAT_PERK_2":"5003" ?
HQ_KILLED": "0",HQ_TAKEDOWNS": "0" ?
"KEYSTONE_ID": "8112", ?

Thank you.

**Comment by fraxiinus** [comment]
unfortunately I don't believe that data (summoner spells) is available

**Comment by fraxiinus** [comment]
SUMMON_SPELL1_CAST
- number of times a summoner spells was used (eg flash)

STAT_PERK_0
- identifier for the runes a player has equipped

HQ_KILLED
- not 100% but likely if they last hit the nexus

HQ_TAKEDOWNS
- not sure, maybe if they helped kill nexus?

KEYSTONE_ID
- the keystone rune equipped, like Dark Harvest or Glacial Augment

**Comment by 1Sam** [comment]
Thank you for your kindness
This is my project... ^___^;;;
https://chuc.shop/Match/ByMatchId/KR_5947008696/Runes

I want to add smite, flash, etc Icons... But I can't get the data from ROFL file... ㅠ.ㅠ

**Reply by fraxiinus** [reply]
For a web application, perhaps the Riot API would be a better fit?
I think they have summoner spell data. https://developer.riotgames.com/

**Reply by 1Sam** [reply]
That are the custom games... So I can't get the data from api server.... ^___^;;;

When I made my mind to make web project the Riot provided custom games... But they stopped providing custom games.
Your project made me to continue my project. Thank you.
